tests/integration-tests/tests/ultraserver/test_gb200.py

```diff
@@ -469,8 +484,11 @@ def test_gb200(
     wait_for_computefleet_changed(cluster, "RUNNING")
 
     # Verify topology plugin is completely disabled after removing force_configuration
-    assert_topology_plugin_completely_disabled(cluster)
+    if instance != "p6e-gb200.36xlarge":
+        assert_topology_plugin_completely_disabled(cluster)
 
     # Verify IMEX still works but topology is completely removed
     assert_imex_healthy(cluster, queue_with_imex, compute_resource_with_imex, max_queue_size_updated)
-    assert_imex_not_configured(cluster, queue_without_imex, compute_resource_without_imex)
+    with soft_assertions():
+        if instance != "p6e-gb200.36xlarge":
```

Comment:
if instance is m6g, which is the one used to simulate b200 instance type, we want imex to be configured.